app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsImageTagsFragment.kt

```diff
@@ -326,7 +326,7 @@ class SuggestedEditsImageTagsFragment : SuggestedEditsItemFragment(), CompoundBu
         publishSuccess = false
 
         funnel?.logSaveAttempt()
-        EditAttemptStepEvent.logSaveAttempt(callback().getLangCode())
+        EditAttemptStepEvent.logSaveAttempt(PageTitle(page!!.title, WikiSite(Service.COMMONS_URL)))
```

Comment:
Is it possible to have a `var pageTitle` for this one? so that we don't have to create `PageTitle` for the following log events.

Reply:
+1 to this too. It can live alongside `page`, and updated whenever `page` is assigned.